### PR TITLE
Targeting changed to only include .NET 6.0 (removed .NET Standard 2.0)

### PR DESCRIPTION
**Pull request by RikuVirtanen**
#9 Implemented changes and targeting to support .NET6 instead of .NET Standard 2.0

### CODE REVIEW

File: Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                return action.Invoke(session);
+            }
+            finally
+            {
+                internalSession?.Dispose();
+            }
+        }
+
+        /// <summary>
+        /// Executes a PowerShell command with parameters, leave parameter value empty for a switch
```
Comment (Svenskapojkarna) [inline Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs]: Please add `[Documentation](link to repo)` so the task will have link to documentation.

File: Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        /// </summary>
```
Comment (Svenskapojkarna) [inline Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs:42]: Also add `<param name="input">documentation</param>` to document input and options parameters.

**Review by Svenskapojkarna — CHANGES_REQUESTED** [review]

File: Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs (hunk 1 above)
Comment (RikuVirtanen) [reply Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs]: Fixed

File: Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs (hunk 2 above)
Comment (RikuVirtanen) [reply Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs:42]: Fixed in both RunCommand and RunScript tasks.

**Pull request merged**